library/macos/FLEMethodChannel.h

```diff
@@ -40,6 +40,12 @@ typedef void (^FLEMethodCallHandler)(FLEMethodCall *_Nonnull call, FLEMethodResu
  */
 extern NSString const *_Nonnull FLEMethodNotImplemented;
 
+/**
+ * A constant that can be passed to an FLEMethodResult to indicate that the message is
+ * missing required arguments.
```

Comment:
The other side of the channel would interpret this as a success callback whose value is the string "Bad Arguments"; see the implementation of the result callback in FLEMethodChannel. What you actually want for cases of bad arguments is to pass an FLEMethodError.